.storybook/main.js

```diff
@@ -10,6 +10,10 @@ module.exports = {
       use: [...tsRule.use, path.resolve(__dirname, `tailwind.${configType.toLowerCase()}.js`)],
     });
 
+    config.resolve.alias['@foxy.io/sdk/core'] = '@foxy.io/sdk/dist/esm/core';
+    config.resolve.alias['@foxy.io/sdk/customer'] = '@foxy.io/sdk/dist/esm/customer';
+    config.resolve.alias['@foxy.io/sdk/integration'] = '@foxy.io/sdk/dist/esm/integration';
```

Comment:
Separate discussion but we'll be changing `Integration` to `Backend`. https://github.com/Foxy/foxy-sdk/issues/7